android/app/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
what is this file

Reply:
I don't know. it was generated automatically. I will try to remove and rebuild